resident/resident-service/src/main/java/io/mosip/resident/dto/RegistrationCenterDto.java

```diff
@@ -56,6 +56,12 @@ public class RegistrationCenterDto extends MasterSyncBaseDto {
 	private String lunchEndTime;
 
 	private Boolean isActive;
+	
+	private String fullAddress;
+	
+	private int serialNumber;
```

Comment:
registration center has serial number?

Reply:
No. Registration centers are not having the serial number. Serial Number is required as part of data generation in PDF.